Algorithmia/datadirectory.py

```diff
@@ -142,3 +143,43 @@ def _iterate_files(self, content):
                 f.set_attributes(file_info)
                 files.append(f)
         return files
+
+
+class LocalDataDirectory():
+    def __init__(self, client, dataUrl):
```

Comment:
Don't need `client` for instantiation.

Reply:
That was done to keep commonality in the API between the LocalDataDirectory and the DataDirectory.  And you could imagine changes we could make where it would be convenient to have access to the client for purposes of creating like files and the like.